parser/internal/runtimeconstants/contants.go

```diff
@@ -0,0 +1,20 @@
+package runtimeconstants
```

Comment:
This file allows the `parseLiteralStruct` function (in the next file) to report fields as being `constants` with this values.

It's a nice simple hack for now (rather than having to actually parse the runtime API's constants to build up this)